### ISSUE TITLE
After unlocking, laptop display has brightness set to 0; external displays retain original brightness

### ISSUE THREAD

**Issue by fluffynuts** [xanderfrangos/twinkle-tray#408]
As per the title: if I lock my machine and walk away, coming back some time later after the screens have blanked, and unlock, the laptop screen comes back with brightness 0 and the other screens remember their brightness settings of 100:

![image](https://user-images.githubusercontent.com/4669340/181749508-82b187df-09a9-48c0-b297-f3d030f341d8.png)

Please let me know if I can supply more info via logs or something similar.

**Comment by xanderfrangos** [comment]
Thanks for the bug report. Just to confirm, you've been setting the laptop's brightness using Twinkle Tray (either using the sliders or hotkeys) and not your laptop's brightness hotkeys?

**Comment by fluffynuts** [comment]
@xanderfrangos correct; before installing Twinkle Tray, I hadn't fiddled with brightness _at all_ on my laptop. A friend recommended the app, and when I was checking it out, I noticed that for some reason, my external displays were not set to brightest, so I'd left it running to set those to 100%. Upon unlocking twice though, I encountered the described situation. I wasn't sure of the cause the first time, which is why I waited for a second occurrence, and then exited Twinkle Tray after that, and the issue didn't happen again.
